crates/bevy_ecs/Cargo.toml

```diff
@@ -32,3 +32,6 @@ downcast-rs = "1.2"
 parking_lot = "0.11"
 rand = "0.8"
 serde = "1"
+
+[dev-dependencies]
+bevy_render = { path = "../bevy_render", version = "0.5.0" }
```

Comment:
A bit of a pickle, I wanted to have a good example that queried something that's easy to grasp, but in order to use Camera I need to add this dependency, if someone has another example Component which does not require this that would be cool :)